src/components/structures/ScrollPanel.tsx

```diff
@@ -226,7 +226,15 @@ export default class ScrollPanel extends React.Component<IProps> {
 
     private onScroll = ev => {
```

Comment:
Let's ship this 🙇 

Merged `develop` so we can ensure it's still good to go test wise even though there are no scroll tests.

Reply:
[TypeScript strict CI job is failing](https://github.com/matrix-org/matrix-react-sdk/actions/runs/3481716623/jobs/5823161509) but it's complaining about an adjacent unrelated line of code and that job is not required to pass.

Cypress flake that passed after the second run ✅

Everything looks okay to merge ⏩